test/common/testlib.py

```diff
@@ -208,17 +208,17 @@ def mouse(self, selector, type, x=0, y=0, btn=0, ctrlKey=False, shiftKey=False,
         self.call_js_func('ph_mouse', selector, type, x, y, btn, ctrlKey, shiftKey, altKey, metaKey)
 
     def click(self, selector):
-        self.mouse(selector + ":not([disabled])", "click", 0, 0, 0)
+        self.mouse(selector + ":not([disabled]):not(.disabled)", "click", 0, 0, 0)
```

Comment:
These changes to testlib seem unrelated. The code changes in this commit don't touch any "disabled" class, and also we really shouldn't use "disabled" classes any more, but use the "disabled" attribute. PR #13061 tries to clean that up (but still needs work). If that is necessary to support our existing code (do the `allowed` changes from above trigger these classes?), please split it out into a separate (previous) commit and describe the reason in the commit message.

Reply:
> do the allowed changes from above trigger these classes?

Yes, exactly. The `update_privileged` pattern uses the `disabled` class to do its business.  This didn't seem to matter before, but now we need to wait also for the .disabled class to go away before clicking on something.

Reply:
In a previous version of this PR, I had changed `update_privileged` to also set the `disabled` attribute, specifically to help the tests.  That however, collides with `users/local.js`, which calls `update_privileged` on some elements and also sets the disabled attribute for the same element.  That was the test failure you had called out earlier.
So I reverted that.
Of course what `users/locals.js` is doing is ugly and deserves cleanup, possibly by rewriting the whole thing in React.  Feels a bit out of scope here.